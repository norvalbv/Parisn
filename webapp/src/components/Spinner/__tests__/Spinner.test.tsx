import { FC } from 'react';
import renderer from 'react-test-renderer';

import Spinner from '../';

describe('<Spinner />', () => {
  test('Spinner props with mandatory props', () => {
    const tree = renderer.create(<Spinner />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
