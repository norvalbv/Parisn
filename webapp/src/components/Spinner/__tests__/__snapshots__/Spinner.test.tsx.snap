// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> Spinner props with mandatory props 1`] = `
<svg
  className="mr-3 -ml-1 h-5 w-5 animate-spin text-black"
  fill="none"
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
>
  <circle
    className="opacity-25"
    cx="12"
    cy="12"
    r="10"
    stroke="currentColor"
    strokeWidth="4"
  />
  <path
    className="opacity-75"
    d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z"
    fill="currentColor"
  />
</svg>
`;
